.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: npm build
```

Comment:
Some better name, e.g. "Publish package to the npm repository"?

Reply:
The idea is to have standard `npm ci` run (which, I believe, can run some tests or, at very least, checks installability of the package) on all builds, and run publish on commits to `master`.

Eventually I wanted to add some kind of version autogeneration, so everything which lands into master will be published automatically to `next`.

Another concern is that having way too long name will be not very comfortable with GH Actions, as it is currently used in UI:

![image](https://user-images.githubusercontent.com/157154/81002813-c063b600-8e41-11ea-9b53-6df68c1d5762.png)

... so probably having 2-3 words is a good idea.

Reply:
@GreyCat Yeah, good point, thanks for the explanation.